homeassistant/components/climate/const.py

```diff
@@ -56,6 +56,12 @@
 # Device is reacting to activity (e.g. movement sensors)
 PRESET_ACTIVITY = "activity"
 
+# Device is in schedule mode
+PRESET_SCHEDULE = "schedule"
```

Comment:
Please keep this in the zha integration. Adding it here would require approval first in an architecture issue.

Reply:
Done.